### PR TITLE
fix: MD033/no-inline-html

### PR DESCRIPTION
**Pull request by nschonni**
Inline HTML get swallowed in MD and HTML rendering

### CODE REVIEW

File: spec/expressions.md [hunk 1]
```diff
@@ -3505,7 +3505,7 @@ bool? operator |(bool? x, bool? y);
 
 The following table lists the results produced by these operators for all combinations of the values `true`, `false`, and `null`.
 
-| `x`     | `y`     | `x & y` | <code>x &#124; y</code> |
+| `x`     | `y`     | `x & y` | `x | y` |
```
Comment (333fred) [inline spec/expressions.md]: This doesn't appear to render correctly.

File: spec/introduction.md [hunk 2]
```diff
@@ -277,13 +277,13 @@ The following table summarizes C#'s operators, listing the operator categories i
 |                                  | `x != y`          | Not equal |
 | Logical AND                      | `x & y`           | Integer bitwise AND, boolean logical AND |
 | Logical XOR                      | `x ^ y`           | Integer bitwise XOR, boolean logical XOR |
-| Logical OR                       | <code>x &#124; y</code> | Integer bitwise OR, boolean logical OR |
+| Logical OR                       | `x | y`           | Integer bitwise OR, boolean logical OR |
```
Comment (333fred) [inline spec/introduction.md]: This doesn't appear to render correctly.

File: spec/introduction.md [hunk 3]
```diff
@@ -277,13 +277,13 @@ The following table summarizes C#'s operators, listing the operator categories i
...
 | Conditional AND                  | `x && y`          | Evaluates `y` only if `x` is `true` |
-| Conditional OR                   | <code>x &#124;&#124; y</code> | Evaluates `y` only if `x` is `false` |
+| Conditional OR                   | `x || y`          | Evaluates `y` only if `x` is `false` |
```
Comment (333fred) [inline spec/introduction.md]: This doesn't appear to render correctly.

File: spec/expressions.md (hunk 1 above)
Comment (nschonni) [reply spec/expressions.md]: Yeah, the Docs preview in VS Code shows it correctly, but GitHub isn't, so I rebased it out

**Pull request merged**